BigQueryStorage/src/V1/Gapic/BigQueryReadGapicClient.php

```diff
@@ -247,15 +247,9 @@ public function createReadSession(array $optionalArgs = [])
             $request->setMaxStreamCount($optionalArgs['maxStreamCount']);
         }
 
-        $descriptorData = [];
-
-        if ($request->getReadSession()) {
-            $descriptorData['read_session.table'] = $request->getReadSession()
-                ->getTable();
-        }
-
-        $requestParams = new RequestParamsHeaderDescriptor($descriptorData);
-
+        $requestParams = new RequestParamsHeaderDescriptor([
+          'read_session.table' => $request->getReadSession()->getTable(),
```

Comment:
Hmm, what happened here? If `$optionalArgs.readSession` isn't set, this will fail.